cabal-testsuite/PackageTests/Backpack/Includes2/cabal-external-target.out

```diff
@@ -1,4 +1,5 @@
 # cabal v2-build
+Querying pkg-config database...
```

Comment:
We could print it only when this step takes a long time (for some definition of long).

Reply:
This a good idea but I think it should be a separate PR :relaxed:

Reply:
What is the problem of touching every testcase? Let me understand, do you find unacceptable to print this message?
It appears only when pkg-config is actually called, which, because of the cache, does not happen at every invocation.

Reply:
> It appears only when pkg-config is actually called, which, because of the cache, does not happen at every invocation.

Yes, but apparently `pkg-config` is called even when no external package is involved, it seems.  Like if I look at a random test, like https://github.com/haskell/cabal/tree/f97fb81a12e0daadc884ca004419d8354de93253/cabal-testsuite/PackageTests/MultiRepl/CabalTooOld , then the sources of this test do not mention anything with `pkg-config`, but still the message appears.

I think `cabal` is already spamming the user with lots of irrelevant information that is printed all the time, and we should strive to reduce this and let cabal communicate the _relevant_ things.

Reply:
Alright, I bumped the message to `info`.